tests/MessagePack.Tests/StandardClassLibraryFormatterTests.cs

```diff
@@ -78,6 +81,28 @@ public void DeserializeByteArrayFromFixArray_Array32()
             Assert.Equal(new byte[] { 1, 2, 3 }, byte_array);
         }
 
+        [Theory]
+        [InlineData(299)]
+        [InlineData(300)]
+        public void BigInteger(int end)
+        {
+            // Test case came from https://github.com/MessagePack-CSharp/MessagePack-CSharp/issues/2060
+            var start = 1;
+
+            var x = ParallelEnumerable
+                .Range(start, end)
+                .Aggregate(
+                    () => System.Numerics.BigInteger.One,
+                    (localProduct, i) => localProduct * i,
+                    (totalProduct, localProduct) => totalProduct * localProduct,
+                    finalProduct => finalProduct);
```

Comment:
Can we avoid factorial computation for our test, as that's irrelevant, and instead just initialize a very large number? Maybe using `Math.Pow(300,300)` or something like that?

Reply:
Yes, I just committed a simpler test case.